docs/standard/base-types/best-practices-strings.md

```diff
@@ -245,7 +245,7 @@ The <xref:System.Char.ToUpper(System.Char)?displayProperty=nameWithType> and <xr
 
 Default interpretation: <xref:System.StringComparison.CurrentCulture?displayProperty=nameWithType>.
 
-By default, both of these methods perform a culture-sensitive comparison.
+By default, both of these methods perform a culture-sensitive comparison. In particular, they may ignore non-printing characters.
```

Comment:
This gets raised as a runtime bug periodically, and closed as "by design", and it remains essentially undocumented unless you downloaded the sorting weights *and* you also intuited that the same sorting weights were used for `StartsWith` and not just for sorting.